Index: src/main/resources/static/dalejs.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/static/dalejs.js b/src/main/resources/static/dalejs.js
--- a/src/main/resources/static/dalejs.js	
+++ b/src/main/resources/static/dalejs.js	(date 1695059326376)
@@ -4,7 +4,7 @@
         const username = document.getElementById("username").value;
         const password = document.getElementById("password").value;
 
-        fetch("/login", {
+        fetch("/registro", {
             method: "POST",
             headers: {
                 "Content-Type": "application/json"
